client/src/components/Modal.jsx

```diff
@@ -50,13 +67,21 @@ class Modal extends React.Component {
 											type="text"
```

Comment:
Check your indentations. This part especially is pretty off.

Reply:
Alright will do!